documentation/resources/api-discovery.md

```diff
@@ -4,140 +4,192 @@ order: 2200
+By listing your API in the API Directory, you also gain access to tools for managing your API consumers through access requests and approvals.
 
-Go to [Gateway Administration](../resources/gateway-admin.md#directory-api) for links to the Directory API.
+## API Listings
 
-## Setup your Draft Dataset
+The API Directory shows available APIs through API listings. Each listing features one API and is visible as a card on the API Directory's main page.
 
-If you do not have a Dataset already defined in the BC Data Catalogue, you can create a draft in the API Services Portal:
+![BC Route Planner API card](assets/api-directory-card.png)
 
-1. Click `Help` in the top right, then `API Docs`
-2. Click the green `Authorize` button, then enter your Client ID and Secret
-3. Click the `PUT /namespaces/{ns}/datasets` accordion item
-4. Click `Try it out`, enter your namespace, personalize any fields in the Request body, remove the lines pertaining to organization, and click `Execute`
-5. Scroll down and ensure a 200 Response was received
+Cards show basic metadata about the API: the API name (`title`), associated ministry (`organization`), description (`notes`), and available environments. This metadata is defined in a `dataset`.
 
+Despite the name `dataset`, don't think that all API's need to serve *data*. Many API's provide services, for example, the [BC Route Planner API](https://api.gov.bc.ca/devportal/api-directory/740?preview=false) returns vehicle routes between start and end points.
+
+!!!info
+Think of `datasets` as API metadata records.
+!!!
+
+Clicking on the API name opens the full API listing details.
+![BC Route Planner API listing details](assets/api-directory-listing.png)
+
+Here, additional metadata details defined in the `dataset` are visible, as well as the associated product(s) which consumers can request access to.
+
+!!!info
+`products` bundle one or more gateway services which are protected in the same way. They are used to manage consumer access.
+!!!
+
+## Sharing Your API
+
+API Listings in the Directory can be created and managed via:
+
+- **Command Line Interface (CLI)**: See the [Quick Start guide](../guides/owner-journey.md) for instructions on downloading the CLI and logging in.
+- **Directory API**: Go to [Gateway Administration](../resources/gateway-admin.md#directory-api) for links to the Directory API specs and Swagger UI.
+- **Web user interface (UI)**: Visit the [API Services Portal](https://api.gov.bc.ca/) and login to the API Provider zone.
+
+!!! API Services Portal web UI limitations
+At this time, not all steps required to create an API listing are possible through the web UI. We recommend using the CLI.
+!!!
+
+### Environments
+
+All API Directory functionality is available in the `test` environment. Feel free to create an API listing there before adding to the production Directory.
```

Comment:
Looking at all our current docs (excluding Owner Journey v1, which should maybe be deprecated) the only places the test env is used are the JWT auth docs and QSG (otherwise it's always tables which also list prod).
Should we be pointing to test more often in our code samples? Or conversely, should we continue not pointing to test (except maybe really basic tutorials like the QSG?)

How about:
```suggestion
All API Directory functionality is accessible in both the `test` and `production` environments. While you are encouraged to utilize the `test` environment for experimentation and training purposes, if you already know the details of the API you're building, you can add it directly to the production Directory.
```
It's a little longwinded but eh. 
Otherwise could cut the second sentence to just:
```
Use the `test` environment to experiment.
```